### PR TITLE
docs: multi-type search

### PR DESCRIPTION
**Pull request by vlcekmi3**
i couldn't find this type of information in documentation, so i made one

### CODE REVIEW

File: Resources/doc/cookbook/multi-type-search.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+Multi type search
+===============
+
+If you want to search not entire index but only some of the types in index.
```
Comment (XWB) [inline Resources/doc/cookbook/multi-type-search.md]: We should rephrase this.

Comment (vlcekmi3) [inline Resources/doc/cookbook/multi-type-search.md]: ok, whats your recommendation?

Searching in subset of types?

**Comment by XWB** [comment]
My English is not perfect, something like 'If you want to search within some index types only'. Perhaps @stloyd can make something of it.

File: Resources/doc/cookbook/multi-type-search.md (hunk 1 above)
Comment (stloyd) [inline Resources/doc/cookbook/multi-type-search.md]: Maybe something like: 
```
If you want to limit your search to just few selected types in given index you could do that this way:

Comment (XWB) [inline Resources/doc/cookbook/multi-type-search.md]: @vlcekmi3 Can you change the text and fix the merge conflict?

Comment (vlcekmi3) [inline Resources/doc/cookbook/multi-type-search.md]: done

**Comment by XWB** [comment]
Thanks :+1:

**Pull request merged**